### PR TITLE
docs/Kbd-theming

### LINKED ISSUE #1075: Theming: Kbd
### Subject

Kbd

### Description

Create a basic theming documentation for the `Kbd` component that showcases how you can style it.

Theming file: [Kbd theming file](https://github.com/chakra-ui/chakra-ui-docs/blob/main/content/docs/components/kbd/theming.mdx)

### Template Reference

For single part components, feel free to use the `Button` component theming doc as a template reference: [Button theming](https://github.com/chakra-ui/chakra-ui-docs/blob/main/content/docs/components/button/theming.mdx).

For multipart components, feel free to use the `Input` component theming doc as a template reference: [Input theming](https://github.com/chakra-ui/chakra-ui-docs/blob/main/content/docs/components/input/theming.mdx).

### PR DESCRIPTION
**Pull request by mahdikhashan**
Closes #1075

## 📝 Description

It adds theming documentations for kbd component.

## ⛳️ Current behavior (updates)

It doesn't have any document right now.

## 🚀 New behavior

It adds a document with working codesandbox example.

### CODE REVIEW

File: content/docs/components/kbd/theming.mdx [hunk 1]
```diff
@@ -2,3 +2,181 @@
...
+> [Component Style](/docs/styled-system/component-style#styling-single-part-components)
+> page.
+
+## Theming properties
+
+The properties that affect the theming of the `Kbd` component are:
+
+- `variant`: The visual variant of the kbd.
+- `colorScheme`: The color scheme of the kbd.
+- `size`: The size of the kbd.
```
Comment (aacevski) [inline content/docs/components/kbd/theming.mdx:19]: We want to accent the Kbd that is a component so let's use the backtics.

```suggestion
- `variant`: The visual variant of the `Kbd`.
- `colorScheme`: The color scheme of the `Kbd`.
- `size`: The size of the `Kbd`.
```

File: content/docs/components/kbd/theming.mdx [hunk 2]
```diff
@@ -2,3 +2,181 @@
 id: kbd
 scope: theming
 ---
+
+The `Kbd` component is a single part component. All of the styling is applied
+directly to the `kbd` element.
```
Comment (aacevski) [inline content/docs/components/kbd/theming.mdx:7]: Capital for the Kbd element.
```suggestion
directly to the `Kbd` element.
```

File: content/docs/components/kbd/theming.mdx [hunk 3]
```diff
@@ -2,3 +2,181 @@
...
+Then, we can use the custom color scale as the color scheme for the kbd:
+
+```jsx live=false
+<Kbd colorScheme='brand'>...</Kbd>
+```
+
+## Changing the default properties
+
+Let's assume we want to change the default size, variant or color scheme of
+every kbd in our app. Here's how we can do that:
```
Comment (aacevski) [inline content/docs/components/kbd/theming.mdx:144]:
```suggestion
every `Kbd` in our app. Here's how we can do that:
```

**Review by aacevski — CHANGES_REQUESTED** [review]
Hey @mahdikhashan, thank you for your contribution to the theming! I just left some small comments regarding typos.

File: content/docs/components/kbd/theming.mdx (hunk 2 above)
Comment (mahdikhashan) [reply content/docs/components/kbd/theming.mdx:7]: hmmm, here I mean <kbd> html element. Do you agree with me to change it to <kbd>?

Comment (aacevski) [reply content/docs/components/kbd/theming.mdx:7]: Ah I see. No need for a change.

**Comment by mahdikhashan** [comment]
@aacevski conflict is solved.

**Pull request merged**